src/lib.rs

```diff
@@ -660,22 +671,22 @@ pub mod pallet {
 		/// - `query_id`: Identifier of reported data.
 		/// - `timestamps`: Batch of timestamps of reported data eligible for reward.
 		#[pallet::call_index(1)]
-		#[pallet::weight(780231000)]
+		#[pallet::weight(<T as Config>::WeightInfo::claim_onetime_tip(T::MaxValueLength::get(), T::MaxClaimTimestamps::get()))]
```

Comment:
Not sure why the first parameter (T::MaxValueLength::get()) is required here? The only variability is based on timestamps and dont think any oracle value is read when claiming timestamps?

Reply:
This is required as internally this extrinsic is looping over submitted reports [here](https://github.com/tellor-io/tellor-pallet/blob/917dbd32ecacdaa1a5a7d7e4d9f35132caac0d8e/src/impls.rs#L675).